src/wallet/rpcwallet.cpp

```diff
@@ -3107,6 +3111,7 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
                     {"hexstring", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The hex string of the raw transaction"},
                     {"options", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED_NAMED_ARG, "for backward compatibility: passing in a true instead of an object will result in {\"includeWatching\":true}",
                         {
+                            {"add_inputs", RPCArg::Type::BOOL, /* default */ "false", "For a transaction with existing inputs, automatically include more if they are not enough."},
```

Comment:
This is backward-incompatible, and it makes no sense for this RPC which does exactly only that? Confused.

Reply:
See @achow101's comment above: https://github.com/bitcoin/bitcoin/pull/16377#issuecomment-517380568

I tend to agree that `walletcreatefundedpspbt` is different from `fundrawtransaction` in this aspect, because `walletcreatefundedpspbt` also sets the destination.

Reply:
I think both `walletcreatefundedpsbt` and `fundrawtransaction` are both most commonly used with no inputs. As I understand it, with `add_inputs` defaulting to false, this behavior doesn't change. Even so, the option is useful in `fundrawtransaction` as sometimes you just want it to set the change output and fee for a transaction after you've already selected the inputs.